games/2048.cpp/Portfile

```diff
@@ -0,0 +1,26 @@
+PortSystem              1.0
+
+PortGroup               github 1.0
+PortGroup               cmake 1.1
+
+github.setup            plibither8 2048.cpp 81c0eebaa2e575c7c7943c01958b88896c9da852
+version                 2019-12-12
+revision                0
+categories              games
+maintainers             {mails.ucas.ac.cn:chenguokai17 @chenguokai} openmaintainer
+
+platforms               darwin
+license                 MIT
+description             Fully featured terminal version of the game "2048" written in C++
+long_description        \
+                        Fully featured terminal version of the game "2048" written in C++ \
+                        which is made to run natively on the GNU/Linux and MacOS platforms.
+
+patchfiles              data_path.diff
+checksums               rmd160  3f1b6aae943e70ac79d107dd899762313ff2ce66 \
+                        sha256  e5746f6505220d62f1c250456731c632d6d2e820f6a66313bc2940dbd2e1dcbd \
+                        size    4205777
+
+cmake.generator         "Unix Makefiles"
```

Comment:
Is this line necessary? Is the default that's set by the cmake 1.1 portgroup not sufficient?

Also, please add the standard modeline to the top of the Portfile.

Reply:
Fixed, thanks:)